### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by ChristianKleineidam** [brendonh/pyth#32]
It would be nice if pyth would support Python 3. Is that support on the roadmap?

**Comment by brendonh** [comment]
I agree it would be nice, but I've never used Py3 and am unlikely to anytime soon. I'd gladly accept a pull request, if it doesn't break Py2.

**Comment by robertour** [comment]
@ChristianKleineidam, check this fork: https://github.com/prechelt/pyth/tree/pyth-py3
